olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -1,115 +1,121 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
It seems like the CRD's version was updated but the CSV still references the [v1alpha1 version](https://github.com/openshift-knative/serverless-operator/blob/master/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L85), which is likely causing the following error:

```
error checking provided apis in bundle serverless-operator.v1.11.0: couldn't find operator.serverless.openshift.io/v1alpha1/KnativeKafka (knativekafkas) in bundle. found: map[operator.knative.dev/v1alpha1/KnativeEventing (knativeeventings):{} operator.knative.dev/v1alpha1/KnativeServing (knativeservings):{}]"
```

Reply:
As of right now, your bundle provides v1 crds and your CSV expects v1alpha1 CRDs to be provided. This applies for both CRDs that you've changed in this PR.

Reply:
thanks for the reply

but...  the actual API version of our bits is `v1alpha1`;

I more here mean about the CRD api itself (using v1 - instead of `v1beta1`). Of the CRD api - not our API

Reply:
My mistake - taking another look.

Reply:
It looks like the V1 yaml is wrong - applying the CRD fails:

```
oc apply -f operator_v1alpha1_knativekafka_crd.yaml 
error: error parsing operator_v1alpha1_knativekafka_crd.yaml: error converting YAML to JSON: yaml: line 60: mapping values are not allowed in this context
```

Reply:
After fixing the indent issue that @markusthoemmes [pointed out](https://github.com/openshift-knative/serverless-operator/pull/607#discussion_r510125665) in the OLM-Catalog CRD I was able to build the bundle correctly.